docs/feature_flags.md

```diff
@@ -40,7 +40,17 @@ with more recent data.
 
 More details can be found [here](querying/basics.md#offset-modifier).
 
-## Remote Write Receiver
+# Evaluation-time-aligned subqueries in PromQL
+
+`--enable-feature=promql-eval-aligned-subqueries`
+
+As opposed to the normal `[x:y]` subquery syntax, which aligns with multiples of the subquery's
+interval as a [performance optimization](https://www.robustperception.io/promql-subqueries-and-alignment),
+the expiremental syntax `[x::y]` aligns with the evaluation time no matter what it is. This gives you
+the benefit of intervals aligned exactly where you'd expect them to be, but could also lead to the
+number of samples needed for the query blowing up, which is why it's disabled by default.
```

Comment:
I think this could be rephrased to be more helpful for someone unfamiliar with the history.
It could give two examples, one with `:` and one with `::`, to illustrate.
"where you'd expect them to be" seems particularly unhelpful.